modules/common/deployment/deployment.go

```diff
@@ -14,14 +14,15 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package common
+package deployment
```

Comment:
We sort of had all the "helper" packages in one `common` package beforehand, right?  It seemed fine that way, but this new way could also work.  I'm ambivalent on this one.

Reply:
One other thought: the new way, for me, makes the code easier to read/locate.  When I see something like `service.GenericServicePort` in some code, I know I can find that code in the `service` package.  Beforehand it might have just been `GenericServicePort` without a package qualifier, which ostensibly makes it harder to locate.